File: CS/Q147080/Tab/ViewInfo/MyOffice2003TabHeaderViewInfo.cs
using DevExpress.XtraTab.ViewInfo;
using DevExpress.XtraTab.Drawing;

namespace DxSample.Tab.ViewInfo {
    public class MyOffice2003TabHeaderViewInfo : Office2003TabHeaderViewInfo {
        public MyOffice2003TabHeaderViewInfo(BaseTabControlViewInfo viewInfo) : base(viewInfo) { }
        private MyTabRowViewInfoCollection rows = new MyTabRowViewInfoCollection();
        public override BaseTabRowViewInfoCollection Rows { get { return rows; } }
        protected override bool UpdateFillPageBounds() { return false; }
    }
}

File: VB/Q147080/Tab/ViewInfo/MyOffice2003TabHeaderViewInfo.vb
Imports DevExpress.XtraTab.ViewInfo
Imports DevExpress.XtraTab.Drawing

Namespace DxSample.Tab.ViewInfo
    Public Class MyOffice2003TabHeaderViewInfo
        Inherits Office2003TabHeaderViewInfo

        Public Sub New(ByVal viewInfo As BaseTabControlViewInfo)
            MyBase.New(viewInfo)
        End Sub

        Private rows_Renamed As New MyTabRowViewInfoCollection()
        Public Overrides ReadOnly Property Rows() As BaseTabRowViewInfoCollection
            Get
                Return rows_Renamed
            End Get
        End Property
        Protected Overrides Function UpdateFillPageBounds() As Boolean
            Return False
        End Function
    End Class
End Namespace
